doc/mgmt/gnmi/SONiC_GNMI_Server_Interface_Design.md

```diff
@@ -456,25 +456,14 @@ The full configuration request will be overwritten by subsequent full configurat
 
 #### 1.2.1.12 Backward Compatibility
 
-SONiC telemetry is using prefix target to identify target database, and we will add a new target to support mixed schema.
-
-    enum Target {
-      option allow_alias = true;
-      APPL_DB         = 0;
-      ASIC_DB         = 1;
-      COUNTERS_DB     = 2;
-      LOGLEVEL_DB     = 3;
-      CONFIG_DB       = 4;
-      // PFC_WD_DB shares the the same db number with FLEX_COUNTER_DB
-      PFC_WD_DB       = 5;
-      FLEX_COUNTER_DB = 5;
-      STATE_DB        = 6;
-      // For none-DB data
-      OTHERS          = 100;
-      // For mixed schema
-      MIXED_SCHEMA    = 101;
-    }
+SONiC telemetry is using empty target to support sonic management framework.
+
+We will below solution to support new GNMI config interface:
+<img src="images/single_target.svg" alt="overwritten-config" width="1000px"/>
```

Comment:
Suggest merge this diagram into original https://github.com/sonic-net/SONiC/blob/master/doc/mgmt/gnmi/images/data_flow.svg
No need add extra diagram.